cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/model/optimized/ConsentStatusApiModelExt.kt

```diff
@@ -31,6 +31,7 @@ internal fun CcpaCS.toCCPAConsentInternal(): CCPAConsentInternal {
         thisContent = JSONObject(),
         signedLspa = signedLspa,
         webConsentPayload = webConsentPayload,
+        uspstring = this.uspstring ?: "1YNN"
```

Comment:
can we use constant here, instead of hardcoded value?

Reply:
makes sense